crawlers/mooncrawl/mooncrawl/cu_reports_crawler/cli.py

```diff
@@ -0,0 +1,616 @@
+    max_retries: int = 20,
+) -> Any:
+
+    """
+    Await the query to be update data on S3 with if_modified_since and return new the data.
+    """
+
+    keep_going = True
+
+    repeat = 0
+
+    if_modified_since_datetime = datetime.datetime.utcnow()
+    if_modified_since = if_modified_since_datetime.strftime("%a, %d %b %Y %H:%M:%S GMT")
+
+    time.sleep(2)
+
+    data_url = client.exec_query(
+        token=token,
+        name=query_name,
+        params=params,
+    )  # S3 presign_url
+
+    while keep_going:
+        time.sleep(time_await)
+        try:
+            data_response = requests.get(
+                data_url.url,
+                headers={"If-Modified-Since": if_modified_since},
+                timeout=5,
+            )
+        except Exception as e:
+            print(e)
+            continue
+
+        if data_response.status_code == 200:
+            break
+
+        repeat += 1
+
+        if repeat == max_retries // 2:
+            data_url = client.exec_query(
+                token=token,
+                name=query_name,
+                params=params,
+            )
+
+        if repeat > max_retries:
+            print("Too many retries")
+            break
+    return data_response.json()
+
+
+def generate_report(
+    client: Moonstream,
+    token: Union[str, UUID],
+    query_name: str,
+    params: Dict[str, Any],
+    bucket_prefix: str,
+    bucket: str,
+    key: str,
+):
+    """
+    Generate the report.
+    """
+
+    try:
+
+        json_data = recive_S3_data_from_query(
+            client=client,
+            token=token,
+            query_name=query_name,
+            params=params,
+        )
+
+        client.upload_query_results(
+            json.dumps(json_data),
+            bucket,
+            f"{bucket_prefix}/{key}",
+        )
+        print(f"https://{bucket}/{bucket_prefix}/{key}")
+    except Exception as err:
+        print(
+            f"Cant recive or load data for s3, for query: {query_name}, bucket: {bucket}, key: {key}. End with error: {err}"
+        )
+
+
+def create_user_query(
+    client: Moonstream,
+    token: Union[str, UUID],
+    query_name: str,
+    query: str,
+):
+    """
+    Create a user query.
+    """
+
+    try:
+        client.create_query(token=token, name=query_name, query=query)
+    except Exception as err:
+        print(f"Cant create user query: {query_name}. End with error: {err}")
+
+
+def delete_user_query(client: Moonstream, token: str, query_name: str):
+    """
+    Delete the user's queries.
+    """
+
+    id = client.delete_query(
+        token=token,
+        name=query_name,
+    )
+
+    print(f"Query with name:{query_name} and id: {id} was deleted")
+
+
+def init_game_bank_queries_handler(args: argparse.Namespace):
+
+    """
+    Create the game bank queries.
+    """
+
+    client = Moonstream()
+
+    for query in cu_bank_queries:
+
+        try:
+            try:
+                # delete
+                delete_user_query(
```

Comment:
Can you please descripbe in func description, why we at first trying to delete user query and if it fails we just continue?

Reply:
Was add `--overwrite` parameter instead, so that logic now be are default. 
Just from logic when you adding more then 1 new queries it more simple to work.